pkg/dataflatten/flatten.go

```diff
@@ -460,6 +460,8 @@ func stringify(data interface{}) (string, error) {
 			return s, nil
 		}
 		return base64.StdEncoding.EncodeToString(v), nil
+	case uint8:
+		return strconv.FormatUint(uint64(v), 10), nil
```

Comment:
We also need to add casing for `uint16` (See several properties from [Win32_LogicalDisk](https://docs.microsoft.com/en-us/windows/win32/cimwin32prov/win32-logicaldisk))

```
	case uint16:
		return strconv.FormatUint(uint64(v), 10), nil
```

Reply:
Good call.